ts/screens/wallet/payment/ConfirmPaymentMethodScreen.tsx

```diff
@@ -251,14 +252,16 @@ const ConfirmPaymentMethodScreen: React.FC<Props> = (props: Props) => {
               <H4>{` ${maybePsp.value.businessName}`}</H4>
             </H4>
           )}
-          <TouchableDefaultOpacity onPress={props.pickPsp}>
-            <Text link={true} bold={true}>
+          <TouchableDefaultOpacity accessible>
```

Comment:
Do you still need the `TouchableDefaultOpacity` now that the press event is handled by the inner `Link`?

Reply:
Fixer PR has suggested ✅